.github/workflows/test-suite.yml

```diff
@@ -3,7 +3,7 @@ name: test-suite
 on:
   push:
     branches:
-      - master
```

Comment:
I assume this isn't intended to get merged into master. Just leaving a note here so we don't forget about it.

Reply:
This is here so that I don't submit PRs with failing tests ;)